lib/oidcstrategy.js

```diff
@@ -863,147 +863,147 @@ Strategy.prototype._authCodeFlowHandler = function authCodeFlowHandler(code, req
+        return next(new InternalOAuthError('failed to obtain access token', getOAuthAccessTokenError));
+      }
 
-        var id_token = params.id_token;
+      var id_token = params.id_token;
 
-        // id_token should be present
-        if (!id_token)
-          return self.fail('id_token is not received');
+      // id_token should be present
+      if (!id_token)
+        return self.fail('id_token is not received');
 
-        // token_type must be 'Bearer'
-        if (params.token_type !== 'Bearer') {
-          log.info('token_type received is: ', params.token_type);
-          return self.fail(`token_type received is not 'Bearer'`);
-        }
+      // token_type must be 'Bearer'
+      if (params.token_type !== 'Bearer') {
+        log.info('token_type received is: ', params.token_type);
+        return self.fail(`token_type received is not 'Bearer'`);
+      }
 
-        log.info('received id_token: ', id_token);
+      log.info('received id_token: ', id_token);
 
-        return self._validateResponse(id_token, null, access_token, req, (jwtClaimsStr, jwtClaims) => {
-          // for 'code id_token' flow, check iss/sub in the id_token from the authorization endpoint
-          // with those in the id_token from token endpoint
-          if (issFromPrevIdToken && issFromPrevIdToken !== jwtClaims.iss)
-            return self.fail('After redeeming the code, iss in id_token from authorize_endpoint does not match iss in id_token from token_endpoint');
-          if (subFromPrevIdToken && subFromPrevIdToken !== jwtClaims.sub)
-            return self.fail('After redeeming the code, iss in id_token from authorize_endpoint does not match iss in id_token from token_endpoint');
+      return self._validateResponse(id_token, null, access_token, req, (jwtClaimsStr, jwtClaims) => {
+        // for 'code id_token' flow, check iss/sub in the id_token from the authorization endpoint
+        // with those in the id_token from token endpoint
+        if (issFromPrevIdToken && issFromPrevIdToken !== jwtClaims.iss)
+          return self.fail('After redeeming the code, iss in id_token from authorize_endpoint does not match iss in id_token from token_endpoint');
+        if (subFromPrevIdToken && subFromPrevIdToken !== jwtClaims.sub)
+          return self.fail('After redeeming the code, iss in id_token from authorize_endpoint does not match iss in id_token from token_endpoint');
 
-          const sub = jwtClaims.sub;
-          const iss = jwtClaims.iss;
+        const sub = jwtClaims.sub;
+        const iss = jwtClaims.iss;
 
-          return self._shouldLoadUserProfile(iss, sub, (shouldLoadUserProfileError, load) => {
-            if (shouldLoadUserProfileError) {
-              return next(shouldLoadUserProfileError);
+        return self._shouldLoadUserProfile(iss, sub, (shouldLoadUserProfileError, load) => {
+          if (shouldLoadUserProfileError) {
+            return next(shouldLoadUserProfileError);
+          }
```

Comment:
I have two other pending PR's on oidcstrategy, I cannot replace the tabs with spaces for this PR since that will cause a big merge conflict. I removed the second commit, I will use a separated PR for the tab replacement after all of these 3 PR's in.